src/main.ts

```diff
@@ -417,6 +417,15 @@ function makeLongUtils(options: Options, bytes: ReturnType<typeof makeByteUtils>
     `
   );
 
+  const longToBigint = conditionalOutput(
+    "longToBigint",
+    code`
+      function longToBigint(long: ${Long}) {
```

Comment:
I was wondering if long.js had a built-in `toBigInt` and didn't find anything, but I did find a github issue where someone linked to their long.js fork that has a `toBigInt`:

https://github.com/consento-org/longfn/blob/main/index.mjs#L654

Wdyt about adapting their `toBigInt` and maybe even `fromBigInt`?

https://github.com/consento-org/longfn/blob/main/index.mjs#L100

Seems like those would surely be faster than going through `toString`s, so seems like a good idea, but also willing to leave it as a follow-up issue if you'd like to ship this as-is.

Reply:
Honestly I was just trying to get it to work correctly with zero effort towards optimizing anything.  This, and a few other things in the timestamps can definitely be improved, but I think eventually protobuf's `Reader` and `Writer` will support bigints and drop `Long`.  Regardless, I think it should be merged now and any improvements be done later.